GVRf/Framework/src/org/gearvrf/GVRCameraRig.java

```diff
@@ -54,10 +57,27 @@
     /** Constructs a camera rig without cameras attached. */
     public GVRCameraRig(GVRContext gvrContext) {
         super(gvrContext, NativeCameraRig.ctor());
+        init(gvrContext);
     }
 
     private GVRCameraRig(GVRContext gvrContext, long ptr) {
         super(gvrContext, ptr);
+        init(gvrContext);
+    }
+
+    /** Constructor helper */
+    private void init(GVRContext gvrContext) {
+        setOwnerObject(new GVRSceneObject(gvrContext));
```

Comment:
I am just thinking if this init() should be protected other than init() since in future if there is any  extension of GVRCameraRig, a protected init() may be more helpful.